web/forms/implementing_simple_validation/my_element.dart

```diff
@@ -1,25 +1,27 @@
 @HtmlImport('my_element.html')
```

Comment:
This example should probably be using a [validator](https://github.com/dart-lang/polymer_elements/blob/master/lib/iron_validator_behavior.dart) element do do the validation, I haven't personally tried doing this but that seems to be the new way. See the docs for [iron-input](https://github.com/dart-lang/polymer_elements/blob/23bc0a89e7e82def61fb0d1bda974d7ef58c2b64/lib/iron_input.dart#L27) for a bit of guidance, although it doesn't seem to have a full example.